### ISSUE TITLE
App stops working on NC26

### ISSUE THREAD

**Issue by marxlaml** [nextcloud/notes-ios#37]
Many reviews on the Apple App Store have reported the app not working on NC26, I have the very same issue.
I can authorize to the NC server, but from that point onward I cannot open any note.
The single exception is when the "Offline Mode" is enabled, which defeats the purpose of the app.

Is this known? And if not what can I do to investigate this further?

**Comment by jasondaigo** [comment]
Never tested before NC26 but I can confirm it doesn’t work right now. After login only error message 
![IMG_3508](https://github.com/nextcloud/notes-ios/assets/29674387/d3e0eecd-a349-4bb7-aa58-5fbd84a778bd)
![IMG_3509](https://github.com/nextcloud/notes-ios/assets/29674387/eb710fcf-527a-4b29-9bd5-b6e5c5df1454)

**Comment by marxlaml** [comment]
I looked a bit further into this:

- When I try to open a note on the iOS app nothing happens (not even an error message)
- Instead I get an error on my Nextcloud server saying "Editor text is unknown"
- This error comes from [the DirectEditing Manager](https://github.com/nextcloud/server/blob/37f387b239d63539e702a07eaa648be3ae3e7302/lib/private/DirectEditing/Manager.php#L179)
- The exception is thrown when an editorId is not found in the list of editors
- The list of editors is empty when called

Also:
- When I test this using the Android app it works flawlessly
- The Android app does not even call that function, which is making me think some kind of workflow is outdated in the iOS app


P.S.: Is this app even mantained anymore? The last commit is 2 months old and the negative reviews on the AppStore are not stopping because of issues such as this one (and apparently more issues which are similar in nature)

**Comment by juliushaertl** [comment]
cc @marinofaggiana This seems to be an issue when the text app is not enabled or available for direct editing, which should be indicated in the capabilities.

**Comment by juliushaertl** [comment]
Question to the reporters, are you using server side encryption with or without a master key?

(ref https://github.com/nextcloud/text/issues/3762#issuecomment-1643647718)

**Comment by marxlaml** [comment]
No server side encryption here (and obviously therefore no master key)

**Comment by jasondaigo** [comment]
Was this supposed to be addressed in any way by todays update ? I installed the update but the error remains.

**Comment by joshtrichards** [comment]
Cleaning up some older issues. Are you still experiencing this behavior?

**Comment by marxlaml** [comment]
No, the app is working fine for me. Since I did not modify anything additional myself, I assume an update took care of it. Closing.
